codecov.yml

```diff
@@ -1,3 +1,6 @@
+fixes:
+  - "io/ballerina/stdlib/ftp/compiler/::./compiler-plugin/src/main/java/io/ballerina/stdlib/ftp/plugin/"
```

Comment:
Is this path fix correct? `compiler/` -> `plugin/` ?

Reply:
Removed the path fix to see the output